pkg/reconciler/vertex/scaling/scaling.go

```diff
@@ -215,12 +215,31 @@ func (s *Scaler) scaleOneVertex(ctx context.Context, key string, worker int) err
 		return fmt.Errorf("failed to get metrics of vertex key %q, %w", key, err)
 	}
 	// Avg rate and pending for autoscaling are both in the map with key "default", see "pkg/metrics/metrics.go".
-	rate, existing := vMetrics.ProcessingRates["default"]
+	var rate float64
```

Comment:
Do not need to touch autoscaling, reduce vertices are excluded for autoscaling.

Reply:
I have modified the VertexMetrics response and was summing up the processingRate values for multiple elements of returned array(multiple pods), since reduce is excluded I will only use the first entry of the array ~ 0th element (vMetrics[0].processingRates)